less/core/nav.less

```diff
@@ -24,16 +24,60 @@ html.is-nav-bottom #wrapper {
   &__inner {
     .l-container-responsive(@max-page-width);
     display: flex;
-    align-items: center;
+    align-items: stretch;
   }
 
-  // Override floats on additional `nav__btn` elements part of plugins
-  // that haven't been updated to work with data-order yet.
-  // Can be removed in time.
   &__btn {
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+
+    // Override floats on additional `nav__btn` elements part of plugins
+    // that haven't been updated to work with data-order yet.
+    // Can be removed in time.
     float: none !important;
   }
 
+  &__btn-label {
+    display: none;
+  }
+
+  &.show-label {
+    .show-nav-label(small);
+    .show-nav-label(medium);
+    .show-nav-label(large);
+  }
+
+  &.has-label-right &__btn-label {
+    margin: 0 0 0 0.5rem;
```

Comment:
Would it make sense to use `(@icon-padding / 2)` here? It's the same value (.5rem) but it will keep the margin relative to the button padding. It's also consistent with the [pageNav](https://github.com/cgkineo/adapt-pageNav/blob/b7cc4a7864b1b3e8ce0a54d1ffe87e3cd3a4efd1/less/pageNav.less#L51) plugin (not a core plugin I know) but maintains consistency across navigation UI.

Reply:
Thanks, both. I like the @icon-padding idea here.

> The 0 from the 0.5 value can also be dropped.
In general, I would be hesitant to change this, but maybe it's personal preference. If this is the convention that's most widely used in Adapt Less, then it makes sense to align with that standard. In VSC, I enable an error for "less.lint.zeroUnits / No unit for zero needed" since I think it makes it more readable. You're far less likely to confuse "0.5" with "5" as opposed to not using a leading zero like ".5"

Reply:
Done. Updated margin syntax and switched to @icon-padding / 2